iModelCore/ECPresentation/Source/Rules/PresentationRuleJsonConstants.h

```diff
@@ -256,6 +256,19 @@
 #define CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_RENDERER                     "renderer"
 #define CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_EDITOR                       "editor"
 #define CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_CATEGORYID                   "categoryId"
+#define CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_TYPE							"type"
+
+#define EC_PRIMITIVE_TYPENAME_BINARY													"binary"
+#define EC_PRIMITIVE_TYPENAME_BOOLEAN													"boolean"
+#define EC_PRIMITIVE_TYPENAME_BOOL														"bool"
+#define EC_PRIMITIVE_TYPENAME_DATETIME													"dateTime"
+#define EC_PRIMITIVE_TYPENAME_DOUBLE													"double"
+#define EC_PRIMITIVE_TYPENAME_INTEGER													"int"
+#define EC_PRIMITIVE_TYPENAME_LONG														"long"
+#define EC_PRIMITIVE_TYPENAME_POINT2D													"point2d"
+#define EC_PRIMITIVE_TYPENAME_POINT3D													"point3d"
+#define EC_PRIMITIVE_TYPENAME_STRING													"string"
+#define EC_PRIMITIVE_TYPENAME_IGEOMETRY													"IGeometry"

```

Comment:
I needed to add `ParsePrimitiveType` function which is now used when dealing with calculated properties. Since this function name is generic, I wanted to add parsing to all types, even the ones which aren't used by calculated properties.